sport/app/football/views/fragments/matchSummary.scala.html

```diff
@@ -35,67 +36,47 @@ <h1 class="u-h">@homeTeam.name v @awayTeam.name</h1>
     </div>
 
     <div class="match-info">
-        @competition.map{ competition => <div class="match-info__competition">@competition.fullName</div>}
         @theMatch.venue.map{ venue => <div class="match-info__venue">@venue.name</div> }
     </div>
+-->
```

Comment:
Shall we delete this rather than comment it out?

Reply:
We shall!